terraform/domains/environment_domains/config/production.tfvars.json

```diff
@@ -0,0 +1,22 @@
+{
+    "hosted_zone": {
+      "claim-funding-for-mentor-training.education.gov.uk": {
+        "front_door_name": "s189p01-ittmsdomains-fd",
+        "resource_group_name": "s189p01-ittmsdomains-rg",
+        "domains": [
+          "apex", "www"
+        ],
+        "cached_paths": [
+          "/assets/*"
+        ],
+        "environment_short": "pd",
+        "origin_hostname": "track-and-pay-production.teacherservices.cloud",
+        "redirect_rules": [
+          {
+            "from-domain": "www",
+            "to-domain": "claim-funding-for-mentor-training.education.gov.uk"
+          }
+        ]
+      }
+    }
+  }
```

Comment:
think you need to add redirect_rules to the main.tf for environment domains, as I don't see a redirect rule in fd
i.e.
redirect_rules      = try(each.value.redirect_rules, [])

Reply:
added